pkg/controller/kubelet-config/kubelet_config_nodes.go

```diff
@@ -186,6 +193,24 @@ func (ctrl *Controller) syncNodeConfigHandler(key string) error {
 	return nil
 }
 
+func (ctrl *Controller) syncConfigNodeStatus(cfg *osev1.Node) error {
+	statusUpdateErr := retry.RetryOnConflict(updateBackoff, func() error {
+		newStatusCondition := apihelpers.NewCondition(cgroupModeCondType, metav1.ConditionTrue, cgroupCondReason, cgroupsV1DeprecationMsg)
+		if len(cfg.Status.Conditions) == 0 || newStatusCondition.Message != cfg.Status.Conditions[len(cfg.Status.Conditions)-1].Message {
```

Comment:
Hmm, I think we use a similar pattern in containerruntimeconfigs for error handling (populate new errors). Does Node Config have any other conditions we populate? If we don't, maybe we can simplify this logic a bit so that if we do add other conditions it's a bit easier to build off of, instead of them trying to re-write over each other.

Also for consistency purposes, we should consider naming the function `syncNodeConfigStatus` to match other functions

Reply:
We don't have another condition at the moment but I thought it would be generic and a good idea to update the status this way as the containerruntimeconfig controller does.